owners/package.json

```diff
@@ -9,7 +9,8 @@
     "node": "^10.11"
   },
   "scripts": {
-    "dev": "nodemon",
```

Comment:
If you're removing the use of nodemon, maybe remove nodemon as a dep?

Reply:
Done!